index.html

```diff
@@ -1,10 +1,35 @@
+<!DOCTYPE html>
 <html lang="en">
 <head>
-    <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Document</title>
+<meta charset="UTF-8">
+<meta name="viewport" content="width=device-width, initial-scale=1.0">
+<title>Intro Page - Top 5 National Parks</title>
+<link rel="stylesheet" href="./assets/style.css">
 </head>
+
 <body>
+  <header>
+    <img src="logo.png(example)" alt="National Park Service Logo">
```

Comment:
I'll add the hyperlink to the National park logo when I add the info about the parks